core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -72,6 +72,7 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
     @Input() charLimit: number;
     @Input() customBlocks = '';
     @Input() content: Content = '';
+    @Input() identifier: string;
```

Comment:
```suggestion
    @Input() contentletIdentifier = '';
```

Because it can get confused with the field identifier or any other.